File: 151004/Bosko/lab2/lab2/DTO/NewsRequestTo.cs
using lab2.DTO.Interface;

namespace lab2.DTO
{
    public class NewsRequestTo(int Id, int? UserId, string? Title, string? Content) : IRequestTo
    {
        public int Id { get; set; } = Id;

        public int? UserId { get; set;} = UserId;

        public string? Title { get; set; } = Title;

        public string? Content { get; set; } = Content;
    }
}

File: 1510031/Shamenov/project/lab2/src/main/java/by/bsuir/dc/lab2/dto/NewsRequestTo.java
package by.bsuir.dc.lab2.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.math.BigInteger;
import java.sql.Date;

@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class NewsRequestTo {

    private BigInteger id = BigInteger.valueOf(-1);

    private BigInteger editorId;

    private String title = "";

    private String content = "";

    private Date created = new Date(System.currentTimeMillis());

    private Date modified = new Date(System.currentTimeMillis());
}
